src/lidar/bg_flood_model.py

```diff
@@ -93,14 +96,14 @@ def model_ouput_to_db(outfile, catchment_boundary):
 
 def river_discharge_info(file):
     """Get the river discharge info. from design hydrographs."""
-    with open(rf"{file}\RiverDis.txt") as file:
+    with open(rf"{file}/RiverDis.txt") as file:
```

Comment:
in future would be good to use pathlib instead of path. I define file using pathlib: `file = pathlib.Path(r"string\to\path")`. You can then use  `with open(file / "RiverDis.txt") as file`. See https://stackoverflow.com/questions/48190959/how-do-i-append-a-string-to-a-path-in-python for an example.

You don't need to do this for this PR, but I will start requiring it for future PRs :)

Reply:
done